flow/table.go

```diff
@@ -349,6 +349,11 @@ func (ft *Table) flowPacketToFlow(packet *Packet, parentUUID string, t int64, L2
 	return flow
 }
 
+// Enable/Disable capturing SYN packets
+func (ft *Table) SetSynCapture(enableSynCapture bool) {
```

Comment:
OK, I removed it.
The bug was that I used two different folders of config, now, everything works well.
Thanks!